### ISSUE TITLE
Update Platform documentation on Credential Service Providers (CSPs)

### ISSUE THREAD

**Issue by claytonzook** [department-of-veterans-affairs/va.gov-team#90108]
### Description

We recently noticed some identity related pages that are out of date

*   DS Logon and MHV Credential Service Providers (CSPs) should be updated for timeline of when VA is deprecating them.
*   Login.gov seems to be missing from this documentation, and should probably be added
*   We should probably remove any specific usage data from all 4 of these pages like that a % of folks sign in with a given credential. For example, DS Logon is NOT 60% currently and it's been trending downward over the past couple years, so that that it makes up the smallest share of sign in for at least the past 1-2 years. Login.gov recently surpassed MHV to become the 2nd most used. To say, it's not static and too big a chore to keep up to date.
*   Our team is happy to work with you on making sure whatever we update this to makes the most sense. We can also have our engineers take a look to make sure the dev info on these pages are currently accurate.

### Relevant URLs

DS Logon page - https://depo-platform-documentation.scrollhelp.site/developer-docs/ds-logon  
MHV page - https://depo-platform-documentation.scrollhelp.site/developer-docs/my-healthe-vet  
ID.me page - [https://depo-platform-documentation.scrollhelp.site/developer-docs/id-me](https://depo-platform-documentation.scrollhelp.site/developer-docs/id-me)  
Login.gov page - needed!! I don't see one!  
 

### Which type of team are you on? (Platform team, VFS team, or Leadership)

Identity team (we're part small "p" platform and part VFS, but not on the big "P" Platform contract anymore)

**Comment by taylojill** [comment]
Hi @claytonzook - thanks for your content suggestions. The Platform Content team will review your request and put it in our backlog to refine for next steps.

**Comment by jknipes** [comment]
@claytonzook while there is not a dedicated login.gov page on Platform Website, [this is one of the pages that comes up upon searching for that term](https://depo-platform-documentation.scrollhelp.site/getting-started/platform-console-login-instructions). Do you feel this is sufficient?

**Comment by claytonzook** [comment]
@jknipes I don't think that page is the same. It looks like the one you linked is under Platform Console information and how to use that tool. The login.gov mention there seems to be only for VFS team members who need to create a personal account for work purposes.  
  
The pages I'd mentioned for the account types (eg, credential services for veteran sign ins into VA.gov), ID.me, MHV, and DS Logon and the missing Login.gov page definitely should all be under External Integrations. They explain information like what these services are, how to integrate those services, where to see outage information, and contacts at that service.  
  
The Login.gove integration started maybe about 2 years ago, so I'm wondering if it got missed to add a page like this for it. If that's the case or if it is missing or got deleted or something by accident, I can tag an engineer on my team if that'd be helpful for getting all the information needed for (re)building this page.

**Comment by jknipes** [comment]
@claytonzook I'm thinking we would want to break this up into individual tickets for each page that requires an update, and then schedule time to work with your team one page at a time

**Comment by jknipes** [comment]
Per [this Slack thread](https://dsva.slack.com/archives/CSFV4QTKN/p1725046965429409), Identity team will update the page

**Comment by jknipes** [comment]
As of 2/14/25 all of these pages have been updated:
DS Logon page - https://depo-platform-documentation.scrollhelp.site/developer-docs/ds-logon
MHV page - https://depo-platform-documentation.scrollhelp.site/developer-docs/my-healthe-vet
ID.me page - https://depo-platform-documentation.scrollhelp.site/developer-docs/id-me
Login.gov page - needed!! I don't see one!

**Comment by jknipes** [comment]
Closing as _Done_.
